runtime/integration-tests/src/sibling.rs

```diff
@@ -311,10 +328,7 @@ impl ShouldExecute for DenyReserveTransferToRelayChain {
 		if matches!(origin, MultiLocation { parents: 1, interior: Here }) &&
 			instructions.iter().any(|inst| matches!(inst, ReserveAssetDeposited { .. }))
 		{
-			log::trace!(
-				target: "xcm::barriers",
-				"Unexpected ReserveAssetDeposited from the relay chain",
-			);
+			println!{"Unexpected ReserveAssetDeposited from the relay chain"};
```

Comment:
Why do we change it away from log::trace?

Reply:
This is to remove the log dependency, as it was only added because of this.
I believe the less dependencies we have, the better for us to manage version ups.
I think a removal of even 1 dependency is a win :raised_hands: